crates/swift-bridge-ir/src/codegen/codegen_tests/vec_codegen_tests.rs

```diff
@@ -544,3 +544,116 @@ void __swift_bridge__$some_function(void* arg);
         .test();
     }
 }
+
+/// Test code generation for Rust function that returns a Vec<T> where T is a primitive Rust type.
+mod extern_swift_fn_return_vec_of_primitive_rust_type {
+    use super::*;
+
+    fn bridge_module_tokens() -> TokenStream {
+        quote! {
+            mod ffi {
+                extern "Swift" {
+                    fn some_function() -> Vec<u8>;
+                }
+            }
+        }
+    }
+
+    fn expected_rust_tokens() -> ExpectedRustTokens {
+        ExpectedRustTokens::Contains(quote! {
+            #[allow(non_snake_case)]
+            mod ffi {
+                pub fn some_function() -> Vec<u8> {
+                    unsafe { *Box::from_raw(unsafe { __swift_bridge__some_function() }) }
+                }
+                extern "C" {
+                    #[link_name = "__swift_bridge__$some_function"]
+                    fn __swift_bridge__some_function() -> *mut Vec<u8>;
+                }
+            }
+        })
+    }
+
+    fn expected_swift_code() -> ExpectedSwiftCode {
+        ExpectedSwiftCode::ContainsAfterTrim(
+            r#"
+@_cdecl("__swift_bridge__$some_function")
+func __swift_bridge__some_function () -> UnsafeMutableRawPointer {
+    { let val = some_function(); val.isOwned = false; return val.ptr }()
+}
+"#,
+        )
+    }
+
+    fn expected_c_header() -> ExpectedCHeader {
+        ExpectedCHeader::ContainsAfterTrim("")
```

Comment:
This will pass for all strings string all strings contain `""`. 

You can use `ExpectedCHeader::ExactAfterTrim("")` here instead.